src/structure_managers/adaptor_neighbour_list.hh

```diff
@@ -797,6 +823,18 @@ namespace rascal {
   template <class... Args>
   void
   AdaptorNeighbourList<ManagerImplementation>::update(Args &&... arguments) {
+    if (sizeof...(arguments) > 0) {
+      // TODO(felix) should not have to assume that the underlying manager is
+      // manager centers.
+      auto && atomic_structure{this->manager->get_atomic_structure()};
+      // if the structure has not changed by more than skin**2
+      if (not atomic_structure.is_identical(std::forward<Args>(arguments)...,
```

Comment:
As I commented in the atomic structure, I think the atom type should be added in the check.

Reply:
Good point, we don't want to be caught off guard by random atom type mutations.

Reply:
is fixed in a different branch. could be added now too

Reply:
No, let's address it separately; this PR is bogged down enough as it is